components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -4,225 +4,9 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
-import {trackEvent} from 'actions/telemetry_actions';
-
 import BlockableLink from 'components/admin_console/blockable_link';
 import AlertBanner from 'components/alert_banner';
 
-import {CloudLinks, CloudProducts} from 'utils/constants';
-import PrivateCloudSvg from 'components/common/svg_images_components/private_cloud_svg';
-import CloudTrialSvg from 'components/common/svg_images_components/cloud_trial_svg';
-
-export const contactSalesCard = (
```

Comment:
I moved this function/component to `components/admin_console/billing/billing_subscriptions/contact_sales_card.tsx` because I found it distracting while I was trying to dig in and understand how code on this page worked. No other changes were made to this component.